tests/e2e/tests/mixer/mixer_test.go

```diff
@@ -1467,7 +1467,7 @@ func visitWithApp(url string, pod string, container string, num int) error {
 // getCheckCacheHits returned the total number of check cache hits in this cluster.
 func getCheckCacheHits(promAPI v1.API, app string) (float64, error) {
 	log.Info("Get number of cached check calls")
-	query := fmt.Sprintf("sum(envoy_http_mixer_filter_total_check_calls{app=\"%s\"})", app)
+	query := fmt.Sprintf("sum(envoy_http_mixer_filter_total_check_calls{app.kubernetes.io/name=\"%s\"})", app)
```

Comment:
please revert this part of the changes. The labels used in queries should not be changed.

/cc @gargnupur